pwiz_tools/Skyline/Model/Import.cs

```diff
@@ -411,6 +419,8 @@ public static bool TryInitFormat(string inputLines, out IFormatProvider provider
             return MassListImporter.IsColumnar(inputLines, out provider, out sep, out var columnTypes);
         }
 
+        public int? LineCountLimit { get; set; } // Process no more than LineCountLimit lines when set

```

Comment:
This never actually gets set anywhere. Is that intentional?

Reply:
Nah, just cruft. Removed, thanks.